docs/Getting_Started.md

```diff
@@ -1,17 +1,57 @@
+    If you have questions or ideas, there are multiple ways to get feedback and help:
+    
+    * Post questions or ideas in our [***discussion boards***](https://github.com/microsoft/electionguard/discussions) 
+    * Join our [monthly ***office hours videoconference***](https://github.com/microsoft/electionguard/discussions/61)
+    * Check out and volunteer to work on issues in the repos with tags of [**good first issue**](https://github.com/microsoft/electionguard-python/labels/good%20first%20issue), [**help wanted**](https://github.com/microsoft/electionguard-cpp/labels/help%20wanted) and, as always, [**documentation**](https://github.com/microsoft/electionguard/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation)
 
-## Language
 
-Pick your language of preference:
+## ElectionGuard Specification
 
-[![](images/python-language.png)](guide/Core_Component.md#python)[![](images/c++-language.png)](guide/Core_Component.md#c++) [![](images/c-language.png)](guide/Core_Component.md#c_1) [![](images/c-sharp-language.png)](guide/Core_Component.md#c_2) [![](images/java.png)](https://github.com/JohnLCaron/electionguard-java)
+The ElectionGuard Specification repo addresses the mathematical and conceptual underpinnings of end-to-end verifiability in a manner that comfortably accommodates post-election audits.
+
+The specification houses the core data constructs, tests, and datasets that we recommend to ensure portability and data integrity.
+
+[:fontawesome-brands-github: **ElectionGuard Specification**](./spec/0.95.0/1_Overview.md)
+## ElectionGuard Python
+
+![Python](images/python-language.png)
+
+The ElectionGuard Python library is the *reference implementation* of the ElectionGuard SDK. It covers the entire suite of functionality and processes necessary to implement an end-to-end verifiable election as part a voting system: 
+
+* Key generation ceremony
+* Ballot encryption 
+* Tally ceremony 
+* Ballot decryption
+
+It is designed to be *portable* over *performant*, *universal* over *Pythonic* (although we do try to adhere to the [Zen of Python](https://www.python.org/dev/peps/pep-0020/)).
+
+Any real-world voting use case will need to incorporate the capabilities of the Python library to run an [end-to-end verifiable election](guide/Verifiable_Election.md) or [post-election audit](../Glossary/#post-election-audit).
+
+[:fontawesome-brands-github: **Source**](https://github.com/microsoft/electionguard-python) | [:fontawesome-brands-python: **Package**](https://pypi.org/project/electionguard/) | [:fontawesome-regular-file: **Documentation**](https://microsoft.github.io/electionguard-python/)
+
+!!! info "Community-sourced Java port"
+    ![Java](images/java.png) If you prefer Java, @JohnLCaron contributed a [**Java port**](https://github.com/JohnLCaron/electionguard-java) of our Python library. Thank you John! 
+    
+    [:fontawesome-brands-github: **Source**](https://github.com/JohnLCaron/electionguard-java)
+## ElectionGuard C++
```

Comment:
Not a fan at all of how this manifests with the image. I think this page could be separate pages but should be seriously rethought. 

![Screen Shot 2021-03-01 at 2 17 48 PM](https://user-images.githubusercontent.com/10125297/109547020-e9bc9500-7a98-11eb-9232-b7100c7ede3d.png)